testing/sdk_tasks.py

```diff
@@ -67,7 +67,7 @@ def fn():
 
 
 def check_tasks_not_updated(service_name, prefix, old_task_ids):
-    sdk_plan.get_deployment_plan(service_name)
+    sdk_plan.wait_for_completed_deployment(service_name)
```

Comment:
You do need to wait for deployment to complete.  The point of this check is to validate that the Scheduler thinks no more work should be accomplished and so isn't planning on updating any tasks.  The "change" here is exactly equivalent behavior as in the past, as `get_deployment_plan` would never return until a `200` was returned, which only occurred on plan completion.